crates/nu-cmd-lang/src/core_commands/continue_.rs

```diff
@@ -21,7 +21,9 @@ impl Command for Continue {
 
     fn extra_usage(&self) -> &str {
         r#"This command is a parser keyword. For details, check:
-  https://www.nushell.sh/book/thinking_in_nu.html"#
+  https://www.nushell.sh/book/thinking_in_nu.html
+
+  WARN: continue should be only used in `while`, `loop`, `for`"#
```

Comment:
I would describe this as (and same for `break`):

> `continue` can only be used in `while`, `loop`, and `for` loops. It can not be used with `each` or other filter commands.

Reply:
Thanks.  It's a better message.